src/Stratis.Bitcoin/AddressIndexing/AddressIndexerData.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+
+namespace Stratis.Bitcoin.AddressIndexing
```

Comment:
Maybe rename this file to be `AddressIndexerTypes`?

Reply:
fixed